odim-controller/helmcharts/csplugin/csplugin-config.yaml

```diff
@@ -0,0 +1,7 @@
+csplugin:

```

Comment:
Content under helmcharts should be changed to reflect how a service is deployed, looks like these are in line with a plugin. Please compare any odim service and make the changes accordingly.